doc/dev_guide/internals/box_protocol.rst

```diff
@@ -20,6 +20,13 @@ The binary protocol provides complete access to Tarantool functionality, includi
     asynchronously via the same connection
 *   response format that supports zero-copy writes
 
+..  note::
+
+    Since version :doc:`2.11.0 </release/2.11.0>`, you can use the :ref:`box.iproto <box_iproto>` submodule to access
+    IPROTO constants and features from Lua. The submodule enables to :ref:`send arbitrary IPROTO packets <reference_lua-box_iproto_send>`
+    over the session's socket and :ref:`override the behavior <reference_lua-box_iproto_override>` for all IPROTO
+    request types (including the :ref:`unknown <box_iproto-unknown>` types).
```

Comment:
> including the :ref:`unknown <box_iproto-unknown>` types

This is not a distinguished request type, but rather a constant for the `box.iproto.override` API that allows setting a handler for incoming requests that have an unknown type (image a Tarantool 3.0 instance accepting a `IPROTO_COOL_NEW_FEATURE` request from a Tarantool 4.0 instance).